src/app/drive/components/FileViewer/FileViewer.tsx

```diff
@@ -363,8 +405,8 @@ const FileViewer = ({
               </Dialog.Title>
```

Comment:
File name does not truncate correctly

<img width="1455" alt="Screenshot 2023-02-23 at 14 35 12" src="https://user-images.githubusercontent.com/34165099/220929350-d1cf4492-2eaf-4f4b-9ae4-71167ed8bbe8.png">